models/tags.js

```diff
@@ -0,0 +1,23 @@
+/**
+ *
+ * Represent the Article Tag model
+ * @param {object} sequelize
+ * @param {object} DataTypes The datatype
+ * @returns {object} - Tags
+ */
+const tags = (sequelize, DataTypes) => {
+  const Tags = sequelize.define('Tags', {
+    tag: DataTypes.STRING
```

Comment:
Nice work @idmega2000. Can you please validate for null and empty values?

Reply:
allowNull is set to false now. Thanks